e2e/README.md

```diff
@@ -7,6 +7,27 @@ These tests are run against public testnets: Ropsten and Rinkeby.
 
 e2e package contains a few utilities which are described in a [godoc](https://godoc.org/github.com/status-im/status-go/e2e).
 
+### Flags
+
+#### 1. `-network`
+The `-network` flag is used to provide either a network id or network name which specifies the ethereum network to use
```

Comment:
I am not sure we should allow it to be of string or int type. Can't we decide on one for simplicity? We use `-networkid` in `statusd`?

Reply:
We had this discussion already. IDs aren't so good to remember or more easy for typos. A follow-up may be that in `statusd` also both is allowed in the future.

Reply:
In any case we need to use `switch` and that's why guys suggested we use both. I don't have a strong opinion on that, it doesn't even introduce any new lines.
So if you're strictly against, let's discuss it.

Reply:
I see. Makes sense 👍

Reply:
It's preferable, to allow string, not alot will remember the network id values themselves and preferably being able to use name is far better. This should remain, i very much prefer it this way. More so, we had a previous discussion about this :).